src/types.ts

```diff
@@ -55,7 +55,8 @@ export type Config = {
         clientId: string,
         domain: string,
         // This will be custom to your application, e.g. 'given_name profile'
-        scopes: string
+        scopes: string,
+        connection?: string

```

Comment:
Should also be documented in `setup.md` with an explanation of where to get this string